src/app.ts

```diff
@@ -52,14 +42,36 @@ app.use(errorHandler);
 
 // the mongodb connection
 const db = mongoose.connection;
+// https://mongoosejs.com/docs/connections.html#connection-events
 db.on('error', (error) => {
-	logger.error('[critical] MongoDB connection error:', error);
+	logger.error('[mongodb] error:', error);
+});
+db.on('reconnectFailed', (error) => {
+	logger.error('[mongodb] reconnectFailed:', error);
+});
+const mongoEvents = [
+	'connecting',
+	'connected',
+	'disconnecting',
+	'disconnected',
```

Comment:
i would add disconnected to be logged as error like reconnectFailed

Reply:
will get logged as error now.